### PR TITLE
Adds bubble gum

### PR DESCRIPTION
**Pull request by Mickyan**
You can buy them from vending machines, they function identical to lollipops but when you put them in your mask slot you get a neat little animation for making bubbles

There's three types: Regular, Nicotine and Happiness

I had to repath lollipops to not spaghettify this, pardon my dust. 
There's also a few fluff changes to the Happiness drug in preparation for something later on.

:cl:
add: Added bubble gum! Available in three- wait, two flavors! Regular, and nicotine flavored. Put it your mask slot to make bubbles!
/:cl:

ezgif com-video-to-gif

### CODE REVIEW

**Comment by improvedname** [comment]
BUBBLEGUM DISPOSALS

**Comment by Fikou** [comment]
should have a cursed variant that causes the bubblegum hallucinations like hecksuit

**Comment by RaveRadbury** [comment]
There's a consistency issue with the borg gumball. Do you plan on addressing it in this PR?

**Comment by Mickyan** [comment]
what kind of consistency issue? gumballs are an entirely different type from lollipops anyway

**Comment by RaveRadbury** [comment]
Can you use the borg gumballs like you can use this new gum? If so I misread your changes.

**Comment by Mickyan** [comment]
red label
ezgif com-video-to-gif

@RaveRadbury no the gum just works the same way as lollipops do when you put them in your mask slot

**Comment by evilcyborgmrt** [comment]
Would be great if we could inject chems like with rollies and ecigs. Meth chewing gum anybody?

File: code/modules/food_and_drinks/food/snacks_other.dm [hunk 1]
```diff
@@ -420,29 +420,14 @@
...
-	icon = 'icons/obj/lollipop.dmi'
-	icon_state = "lollipop_stick"
-	item_state = "lollipop_stick"
+/obj/item/reagent_containers/food/snacks/chewable
 	slot_flags = ITEM_SLOT_MASK
-	list_reagents = list(/datum/reagent/consumable/nutriment = 1, /datum/reagent/consumable/nutriment/vitamin = 1, /datum/reagent/iron = 10, /datum/reagent/consumable/sugar = 5, /datum/reagent/medicine/omnizine = 2)	//Honk
-	var/mutable_appearance/head
-	var/headcolor = rgb(0, 0, 0)
 	var/succ_dur = 180
 	var/succ_int = 100
```
Comment (ShizCalev) [inline code/modules/food_and_drinks/food/snacks_other.dm]: mind documenting what these do while you're at it?

Comment (ShizCalev) [inline code/modules/food_and_drinks/food/snacks_other.dm]: mind documenting what this means while you're at it?

Comment (Mickyan) [reply code/modules/food_and_drinks/food/snacks_other.dm]: done 👍

**Review by ExcessiveUseOfCobblestone — COMMENTED** [review]
new/type/path >>> new /type/path

**Comment by RaveRadbury** [comment]
If you're going to add gum that can be equipped to the mask slot can you please update the gumball to allow for the same behavior. This is the consistency issue.

**Comment by Mickyan** [comment]
Rather not bundle more unrelated changes to this, it's an easy change if someone wants to do that later

**Comment by RaveRadbury** [comment]
It's definitely within scope to adjust the gumball, you adjusted the sucker for this PR.

**Comment by gitbirb** [comment]
it'd be nice if it took forever to chew and finish like Popcorn, but gave very little food

**Comment by cacogen** [comment]
clown trunk, harmonica/cassette tape,

**Pull request merged**